### ISSUE TITLE
ACL 2022まとめ

### ISSUE THREAD

**Issue by Sosuke115** [Sosuke115/paper-reading#54]
## リンク

[ACL anthology](https://aclanthology.org/events/acl-2022/)

## 概要

興味ありそうなACL2022の論文をテーマ別にまとめていく

## Sentence embedding

- LaBSE
- [A Sentence is Worth 128 Pseudo Tokens: A Semantic-Aware Contrastive Learning Framework for Sentence Embeddings](https://aclanthology.org/2022.findings-acl.22.pdf)
sentence embeddingをpseudo-token spaceに写像することで**文の長さや文法などの表層的な特徴に依存しない**ベクトルの学習が可能。STSやalignment、uniformityで良い結果。

- [A Contrastive Framework for Learning Sentence Representations from Pairwise and Triple-wise Perspective in Angular Space](https://aclanthology.org/2022.acl-long.336v2.pdf)
既存のSimCSEなどの教師なし文埋め込みは1. シンプルなnormalized temperature-scaled crossentropy loss（NT-Xent）
を利用していて目的関数を探求していない 2. 文間の類似性の程度を考慮していない
そこで論文では以下のNT-Xentにdecision marginを設けたArcCon Lossと、20%, 40%の連続部分語のmaskをかけたs' s''を用いたtriplet lossを提案
![スクリーンショット 2022-10-08 17 29 06](https://user-images.githubusercontent.com/44390274/194698123-54eaa792-faec-44bb-a243-c987b5009079.png)
![スクリーンショット 2022-10-08 17 29 16](https://user-images.githubusercontent.com/44390274/194698135-4235caeb-c8ec-4701-b4ce-d2374cbcbfb5.png)
![スクリーンショット 2022-10-08 17 29 24](https://user-images.githubusercontent.com/44390274/194698140-1e79fd42-a320-4983-b26a-a25cb29889c4.png)

## Multilinguality

[Overlap-based Vocabulary Generation Improves Cross-lingual Transfer Among Related Languages](https://aclanthology.org/2022.acl-long.18.pdf)
低資源言語と関連する高資源言語の語彙の重なりを考慮したOverlap BPEを提案し、Zero-shot cross-lingual transferの性能を向上。

## Language Model and PLMs

- [Deduplicating Training Data Makes Language Models Better](https://aclanthology.org/2022.acl-long.577.pdf)
言語モデルの学習データの重複文を減らす手法を提案。重複を削除してもperplexityは上がらず学習効率が良くなる。下流タスクでの評価はなし。

- [BitFit: Simple Parameter-efficient Fine-tuning for Transformer-based Masked Language-models](https://aclanthology.org/2022.acl-short.1.pdf)
BERTでタスクごとにバイアス項のみのパラメータ調整で、通常のfine-tuningと同等の性能を発揮。

- [PPT: Pre-trained Prompt Tuning for Few-shot Learning](https://aclanthology.org/2022.acl-long.576.pdf)
Prompt tuningの良い初期soft promptを得るために事前学習時にsoft promptを入れる手法を提案。
Prompt tuningについては以下の図を参照
![Screen Shot 2022-06-14 at 9 38 42](https://user-images.githubusercontent.com/44390274/173469727-e24e994a-0ea0-4e05-b6f1-9cdfbb354f74.png)

- [Knowledge Neurons in Pretrained Transformers](https://aclanthology.org/2022.acl-long.581.pdf)
どのようにして知識がTransformerに集積しているのかの説明手法を提案。特にFFN中にあるニューロンのうち、知識を表現するようなKnowledge neuronsの概念を提案し、これについて分析。

- [bert2BERT: Towards Reusable Pretrained Language Models](https://aclanthology.org/2022.acl-long.151.pdf)
小さいPLMを活用して効率的に巨大なPLMを事前学習する手法を提案。
